glide.lock

```diff
@@ -31,4 +33,8 @@ imports:
   version: 7ce08ca145dbe0e66a127c447b80ee7914f3e4f9
   subpackages:
   - errorutil
+- name: golang.org/x/net
+  version: d379faa25cbdc04d653984913a2ceb43b0bc46d7
+  subpackages:
+  - context
```

Comment:
Please don't do that. Currently on openSUSE we haven't switched to Go 1.8 because of issues building on `aarch64` (amongst other things). So using `import "context"` breaks for non-`x86_64` architectures.